examples/chip-tool/commands/clusters/Commands.h

```diff
@@ -1095,6 +1095,7 @@ static void OnThreadNetworkDiagnosticsActiveNetworkFaultsListListAttributeRespon
 | AudioOutput                                                         | 0x050B |
 | BarrierControl                                                      | 0x0103 |
 | Basic                                                               | 0x0028 |
+| Basic                                                               | 0x0028 |
```

Comment:
Duplicate line?

Reply:
Those are created files from zap. The zap has changes which enabled Basic Server. Unfortunately in the controller.zap, The Python end-device should actually use a client.zap. But I did not want to put my fingers into the build side of things and switching to a different zap. If someone could propose a gn mechanism to switch zap files, that would be helpful.